src/services/accounts/AccountsAssetsService.ts

```diff
@@ -10,6 +11,27 @@ import {
 } from '../../types/responses';
 import { AbstractService } from '../AbstractService';
 
+/**
+ * This is a type that is necessary for any runtime pre 9160. It excludes the
+ * `reason` field which 9160 introduces via the following PR.
+ * https://github.com/paritytech/substrate/pull/10382/files#diff-9acae09f48474b7f0b96e7a3d66644e0ce5179464cbb0e00671ad09aa3f73a5fR88
+ *
+ * Both `PalletAssetsAssetBalance`, and `LegacyPalletAssetsAssetBalance` have either,
+ * a `sufficient` or `isSufficient` field exposed instead.
+ */
```

Comment:
BEcause this talks about being the type needed pre-9160, just checking; should it be on the `LegacyPalletAssetsAssetBalance` type?

Reply:
So the issue is they are both Legacy types, one is just way older than the other. 

Not sure what the best approach is for naming, but since `PalletAssetsAssetBalance` is not technically legacy yet because 0.9.16 is not active I chose this naming

Reply:
Oh, sorry, I didn't mean to imply that the type names were wrong; it just sounded like the comment related more to the other one, but I'm easy either way :)

Reply:
You make a great point though, and I think I will update the docs for sure. It really does sound like one is a legacy over the other.

Reply:
Updated :)